### PR TITLE
Removed isPostgresql() check since we support only PostegreSQL

### PR DESCRIPTION
**Pull request by mackdk**
## What does this PR change?

This commit remove the usage of `isPostgresql()` method.

## GUI diff

No difference.

- [X] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [X] **DONE**

## Test coverage
- No tests: already covered

- [X] **DONE**

## Links

- [X] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [X] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: java/code/src/com/suse/manager/webui/utils/MinionActionUtils.java [hunk 1]
```diff
@@ -73,7 +63,7 @@
     private static final Logger LOG = Logger.getLogger(MinionActionUtils.class);
 
     /** Whether the current database is Postgres. */
-    public static final boolean POSTGRES = ConfigDefaults.get().isPostgresql();
+    public static final boolean POSTGRES = true;
```
Comment (mcalmer) [inline java/code/src/com/suse/manager/webui/utils/MinionActionUtils.java:66]: Is this still somewhere used or can we remove that constant as well?

**Review by mcalmer — COMMENTED** [review]
Looks good so far, but I think others should have a look as well.

File: java/code/src/com/suse/manager/webui/utils/MinionActionUtils.java (hunk 1 above)
Comment (mackdk) [reply java/code/src/com/suse/manager/webui/utils/MinionActionUtils.java:66]: Oh yes, it should be removed as well.

**Comment by sbluhm** [comment]
Doesn't this close the door for future different DB support like MariaDB/MySQL?

**Comment by mcalmer** [comment]
> Doesn't this close the door for future different DB support like MariaDB/MySQL?

Yes, but Uyuni has so many very special needs, that I do not see any chance to support other databases at all.
Even bringing Oracle support back would be challeging as we already now use features which are not supported by Oracle.
MariaDB/MySQL are are not an option at all as a lot of basics just do not work.

**Comment by mackdk** [comment]
> Doesn't this close the door for future different DB support like MariaDB/MySQL?

> I do not see any chance to support other databases at all.

Moreover, from what I see, we have already far more places where the code and the queries are postgres-specific, than the ones currently covered by the `isPostgres()` method or the query suffix mechanism. So, even if we wanted to add support for another database, the code this commit removes is misleading at best, if not already outdated.

**Comment by sbluhm** [comment]
ok thanks. will not open that door in future then....

**Review by lucidd — APPROVED** [review]

**Review by mcalmer — APPROVED** [review]

**Pull request merged**